src/App.js

```diff
@@ -4,20 +4,20 @@ import {
   Routes,
   Route,
 } from "react-router-dom";
-import Home from "./pages/Home";
-import SimList from "./pages/SimList"
-import Unspecified from "./pages/Unspecified"
+import Home from "./pages/Home.js";
```

Comment:
I don't know if this is necessary, but if the application works I guess both versions should work